op-challenger/fault/agent.go

```diff
@@ -10,25 +10,16 @@ import (
 	"github.com/ethereum/go-ethereum/log"
 )
 
-// Responder takes a response action & executes.
-// For full op-challenger this means executing the transaction on chain.
-type Responder interface {
-	CanResolve(ctx context.Context) bool
-	Resolve(ctx context.Context) error
-	Respond(ctx context.Context, response types.Claim) error
-	Step(ctx context.Context, stepData types.StepCallData) error
-}
```

Comment:
hmm, I suspect this should stay here given the go convention of declaring interfaces where they're used (the class implementing them doesn't need to reference the interface at all).